packages/backend-modules/republik-crowdfundings/graphql/resolvers/_mutations/submitPledge.js

```diff
@@ -234,15 +234,6 @@ module.exports = async (_, args, context) => {
     // calculate donation
     const pledgeRegularTotal = regularTotal(pledgeOptions, packageOptions)
     const donation = pledge.total - pledgeRegularTotal
-    // check reason
```

Comment:
So for a usual membership for reduced price no reason has to be given as well?

Reply:
Yes. Can but must not.